catboost/private/libs/algo/leafwise_scoring.cpp

```diff
@@ -474,7 +538,11 @@ static TVector<TVector<double>> CalcScoresForOneCandidateImpl(
                     initialFold,
                     leafs,
                     ctx,
-                    &scoreCalcer);
+                    &scoreCalcer, /*statsPtr, parentStatsPtr, siblingStatsPtr);*/
+                    statsPtr != nullptr ? statsPtr + subCandId*maxBucketCount : nullptr,
```

Comment:
we put white-space around all binary operators

Reply:
done